management/commands/makemessages.py

```diff
@@ -17,9 +17,11 @@ def add_arguments(self, parser):
         super(Command, self).add_arguments(parser)
 
         parser.add_argument('--yes-location', action='store_true', dest='yes_location',
-                            default=False, help="Do write '#: filename:line' lines.")
+                            default=settings.ADD_LOCATION if hasattr(settings,'ADD_LOCATION') else False, 
```

Comment:
This should work fine, looks good.

PS. a minor improvement would be using this
```
getattr(object, name[, default])
Return the value of the named attribute of object. name must be a string. If the string is the name of one of the object’s attributes, the result is the value of that attribute. For example, getattr(x, 'foobar') is equivalent to x.foobar. If the named attribute does not exist, default is returned if provided, otherwise AttributeError is raised.
```

Reply:
agreed!

Reply:
Agreed!